gauss_imputation_known_params_demo.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    gen_imputer_data()
</pre><p>Since both the x-axis and y-axis are fixed. Let us have y-label only for the first subplot and remove y-label and y-ticklabels from other two. You can specify these details via a dictionary input to the function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
@patel-zeel So the x-axis will still be visible for the other 2 correct?

Reply:
Yes

Reply:
Done